### PR TITLE
Admin can remove users from appointments

### LINKED ISSUE #64: Admins should remove registered user in appointment
Copied from https://github.com/Sirimangalo/meditation-plus-server/issues/8.

> I should be able to remove anyone's appointment, maybe simply by clicking on it.

### PR DESCRIPTION
**Pull request by olastor**
Fixes #64

### CODE REVIEW

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export const routes: RouterConfig = [
       { path: 'appointments', component: AppointmentAdminComponent },
       { path: 'appointments/new', component: AppointmentFormComponent },
       { path: 'appointments/:id', component: AppointmentFormComponent },
+      { path: 'appointments/remove/:id', component: AppointmentFormComponent },
```
Comment (binarious) [inline src/app/app.routes.ts]: This isn't used. Please remove.

Comment (olastor) [inline src/app/app.routes.ts]: It is [here](https://github.com/Sirimangalo/meditation-plus-server/pull/19/files#diff-e3f6107adee860929b706ea0a7b7f2d6R177) and [here](https://github.com/Sirimangalo/meditation-plus-angular/blob/0b32e07d4c72a91394d3368b3e59dbedbcc201f8/src/app/appointment/appointment.service.ts#L70)??

Comment (binarious) [inline src/app/app.routes.ts]: These are server routes. The ones in `app.routes.ts` are client routes.

Comment (olastor) [inline src/app/app.routes.ts]: Ahh, got it

**Pull request merged**